src/pybamm_cookiecutter/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Copyright (c) 2023 Agriya Khetarpal. All rights reserved.
+
+pybamm-cookiecutter: A template for creating battery modeling projects based on PyBaMM
+"""
+
+
+__version__ = "0.1.0"
```

Comment:
Let's start with this

```suggestion
__version__ = "0.0.1"
```

Reply:
till the time we switch to VCS versioning ;)